src/xrGame/Level_input.cpp

```diff
@@ -112,7 +112,7 @@ extern float g_separate_radius;
 #include "xrScriptEngine/script_engine.hpp"
 #include "ai_space.h"
 
-void CLevel::IR_OnKeyboardPress(int key)
```

Comment:
Please, revert this `key -> dik` and another related changes. DIK stands for Direct Input Key, but we don't use DirectInput anymore. And this change just increases diff size without adding something useful.

Reply:
I resolved.